client/lib/voting.ts

```diff
@@ -27,6 +27,8 @@ export const voteFormSchema = makeRequired(
   requiredFields
 );
 
+export const voteUIFormSchema = voteUISchema;
```

Comment:
I don't understand this line. Why are you exporting it with a different name? Why can't the adjacent file just import voteUISchema?

Reply:
This is for consistency so that all the schemas are available from `voting.ts`. The name change is again for consistency with the previous schema. I could also import it from the higher-level component, but the other schema is imported from this file so that would add some unnecessary overhead and codebase design complications.

Reply:
Can we just put the definition of the `voteUISchema` in voting.ts? Sorry for being nitpicky lol

Reply:
Sure.